src/main/java/core/basesyntax/service/impl/operation/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.service.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public int executeOperation(int initBalance, int quantity) {
```

Comment:
The `executeOperation` method correctly adds the quantity to the initial balance, which is the expected behavior for a return operation. Make sure to validate the input parameters for any possible invalid values before performing the operation.

Reply:
++